modules/poop/index.js

```diff
@@ -127,6 +135,11 @@ module.exports = class Poop extends BaseStorageModule {
     return ['poop-board', 'poopboard', ':poop:', 'poops'];
   }
 
+  isBlackListed(channel) {
+    let channels = ['announcements', 'incident-discussion', 'product-chat']
+    return channels.indexOf(channel) !== -1
```

Comment:
Let's use something modern like `includes` to check in an array.

Reply:
done, though `indexOf` still is a fun javascriptism